### PR TITLE
Base URL to download Helm Client has changed to https://get.helm.sh

### PR DESCRIPTION
**Pull request by mathieu-benoit**
https://github.com/microsoft/azure-pipelines-tasks/issues/10727

And changed the `stableHelmVersion` to `v2.14.1`.

### CODE REVIEW

File: Tasks/HelmInstallerV0/task.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 0,
         "Minor": 153,
```
Comment (bansalaseem) [inline Tasks/HelmInstallerV0/task.json]: please change M153 to M154 and Patch to 1 as we have entered 154 sprint

Comment (mathieu-benoit) [reply Tasks/HelmInstallerV0/task.json]: Got, it, done.

**Comment by ds-ms** [comment]
@mathieu-benoit Thank you for the contribution. :)
It's almost there. 

The same logic needs to be updated in the [helmutility](https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/Common/kubernetes-common-v2/helmutility.ts#L52) in `kubernetes-common-v2` as well; this is being consumed by the `HelmInstallerV1` task. It uses old URLs for downloading.

**Comment by mathieu-benoit** [comment]
Oh ok... what about https://github.com/microsoft/azure-pipelines-tasks/blob/master/Tasks/Common/kubernetes-common/helmutility.ts#L52 as well?

**Comment by ds-ms** [comment]
You don't need to update `kubernetes-common` because the references don't use any helm functionalities. Also, we'll be deprecating it soon.

**Comment by ds-ms** [comment]
Given that you're touching the module from Common folder, you need to bump up the task versions of the dependent tasks. 
In your case: `HelmDeployV0, HelmInstallerV1, KubectlInstallerV0, KubernetesV0, KubernetesV1, KubernetesManifestV0` tasks.

**Comment by mathieu-benoit** [comment]
@ds-ms, it looks it's already done here: https://github.com/mathieu-benoit/vsts-tasks/commit/4c0a605af6fa651b9b7cfed9f958a828a1e68121?

**Comment by ds-ms** [comment]
This would be required every time you make an update. You can see the tests failing in all your checks.

**Pull request merged**